common/content/statusline.js

```diff
@@ -148,13 +148,14 @@ const StatusLine = Module("statusline", {
         var toggle_off = function () {
             bb.style.height = '0px';
             bb.style.overflow = 'hidden';
+            sv.contentSeparator = highlight.get('ContentSeparator').value;
```

Comment:
This is because the `contentseparator` could have changed or loaded after line 133 was run?